crates/biome_cli/tests/commands/lint.rs

```diff
@@ -1047,7 +1047,7 @@ fn fs_files_ignore_symlink() {
         check_windows_symlink!(symlink_dir(&testcase2_path, symlink_testcase2_path));
     }
 
-    let config_path = root_path.join("rome.json");
```

Comment:
I do remember that I left some test with `rome.json` to verify the warning. Can we keep it?